modules/data.land/R/ISCN_extract.R

```diff
@@ -0,0 +1,31 @@
+#' Extract ISCN SOC initial conditions from existing ISCN database.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param ens ensemble number.
+#'
+#' @return A data frame containing sampled SOC, each row represent each site.
+#' @export
+#' 
+#' @examples
+#' @author Dongchen Zhang
+#' @importFrom magrittr %>%
```

Comment:
Is there a reason you need to use magrittr pipes over R native pipes |> ?  If yes, I think I saw recent PRs where @infotroph imported from dplyr instead of magrittr (presumably to reduce dependencies?)

Reply:
@mdietze you're right that I switched data.atmosphere to import from dplyr, and yes it was to reduce dependencies, but looks like the existing pipe imports in data.land are still from magrittr. I support keeping this one from magrittr and switching in a separate PR.

As for |> vs %>%, I've still been using %>% in packages that are already using it, just for consistency. I don't object to switching to |> in new code -- when we do start using native pipes in a given package, we should add `Depends: R (>= 4.1)` to its DESCRIPTION.

Reply:
Reverted back to `dplyr`